scylla-server-rust/src/error.rs

```diff
@@ -10,30 +10,49 @@ use tracing::warn;
 
 pub enum ScyllaError {
     PrismaError(QueryError),
-    NotFound,
+    /// Not available in mock mode, which the system is in
+    NotProd(String),
+    /// An instruction was not encodable
+    ImpossibleEncoding(String),
+    /// Could not communicate to car
+    CommFailure(String),
+    /// A query turned up empty that should not have
+    EmptyResult,
```

Comment:
put comments to the right instead of above each enum name, looks cleaner

Reply:
Alright.  I don't think rust docs will pick up the comment however.

Reply:
Ya in fact rust will still count the comment as the item below the comment leading it to be confusing.

Reply:
Ugh, that is dumb as hell, makes sense tho